### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,73 @@
...
+
+- **Test the components and provide feedback**: We would love to hear your feedback on anything related to the components, like features, API and design. The best way to start is by [trying them out](https://vaadin.com/components/browse). And to get a quick response, either drop a question/comment on the chat or open an issue in GitHub.
+- **Report bugs**: File issues for the components in their respective GitHub projects. You can find links to them from the list in this readme.
+- **Send pull requests**: If you want to contribute code, check out the development instructions below.
+
+We encourage you to read the [contribution instructions by GitHub](https://guides.github.com/activities/contributing-to-open-source/#contributing) also.
+
+## Development
+
+If you want to develop the components yourself see the [development instructions](DEVELOPMENT.md) that apply to all Vaadin components.
```
Comment (web-padawan) [inline README.md:69]: Please include the `DEVELOPMENT.md` to this PR

Comment (tomivirkki) [reply README.md:69]: Ping @Artur-

Comment (Artur-) [reply README.md:69]: Done

**Comment by web-padawan** [comment]
LGTM

**Pull request merged**